### ISSUE TITLE
Constructor parameter should be object.

### ISSUE THREAD

**Issue by amoretspero** [amoretspero/periodic-dataloader#2]
When things get complicated, object parameter will help. Specifically when providing only some of optional parameters.
  
```typescript
// Creates unique enabled loader.
const pdl = new PeriodicDataLoader(100, loadFn, false, undefined, true);
```  
Below will be more elegant than above.  
```typescript
const pdl = new PeriodicDataLoader({ interval: 100, loadFn, unique: true });
```

**Comment by amoretspero** [comment]
Working on this. Should be published with v0.2.0.

**Comment by amoretspero** [comment]
Fixed in commit https://github.com/amoretspero/periodic-dataloader/commit/9570507a68a839e28e812658384e2bc9d3c57afb
